app/views/content/tips-on-applying-for-teacher-training.md

```diff
@@ -145,12 +144,14 @@ Once you've received 2 or more references, you can select the 2 you want to incl
 If you’re struggling to choose your referees, [get free one-on-one support from a
 teacher training adviser](/tta-service).
 
-## Your personal statement
+## Write your teacher training personal statement
```

Comment:
```suggestion
## Writing your teacher training personal statement
```

Reply:
@EllieNodder I changed the H2 from 'Writing your teacher training personal statement' to 'Write your teacher training personal statement' to make it consistent with the other titles on Tips

However, I'm now changing it back again to 'Writing.....' as otherwise the internal links you've created will break!